library/src/app/components/account/account-details/account-details.component.ts

```diff
@@ -37,14 +37,17 @@ import {
   SymbolPriceBankModel,
```

Comment:
Does it make sense to have an account details component for trades and transfers separately instead of having them both in the same component?